src/py-opentimelineio/opentimelineio/url_utils.py

```diff
@@ -4,12 +4,14 @@
 """Utilities for conversion between urls and file paths"""
 
 import os
+import re
```

Comment:
```suggestion
```

The unused "re" import stopped the CI.

Reply:
Sure can do! Had no idea the CI was so thorough!